pkg/v16/resource/clusterconfigmap/desired_test.go

```diff
@@ -46,7 +47,7 @@ func Test_Resource_GetDesiredState(t *testing.T) {
 					},
 				},
 				Data: map[string]string{
-					"baseDomain": "giantswarm.io",
+					"values": "baseDomain: giantswarm.io\n",
```

Comment:
Is it okay? it's not even start with `|` as example shown

Reply:
Yes this will work fine. It's what we do in app-operator in the configmap resource.